### PR TITLE
Raise upper limit of dyn_idle_min_rpm

### PR DESCRIPTION
**Pull request by haslinghuis**
Update after  betaflight/betaflight/pull/11769

Thanks to @KarateBrot for raising the idea.

### CODE REVIEW

**Comment by KarateBrot** [comment]
The original idea is from @sugaarK, I just made the PR for it.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by sugaarK** [comment]
Works good too!

File: src/tabs/pid_tuning.html
Comment (McGiverGim) [inline src/tabs/pid_tuning.html:765]: This does not depend of the Betaflight version?

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: src/tabs/pid_tuning.html
Comment (haslinghuis) [reply src/tabs/pid_tuning.html]: Thanks. Updated. As the PG version in firmware has not been raised we should keep it designate this for 4.4 anyway?

Comment (KarateBrot) [reply src/tabs/pid_tuning.html]: In this case the CLI command `dyn_idle_min_rpm` is backwards compatible cause we're only extending the range, not the underlying data type. So we could add it to 4.3.2.

Comment (haslinghuis) [reply src/tabs/pid_tuning.html]: Thanks. Changed to 4.3

**Review by Asizon — APPROVED** [review]

File: src/js/tabs/pid_tuning.js
Comment (Asizon) [inline src/js/tabs/pid_tuning.js:495]: It should be 1.45??

**Review by Asizon — CHANGES_REQUESTED** [review]

File: src/js/tabs/pid_tuning.js
Comment (haslinghuis) [reply src/js/tabs/pid_tuning.js:495]: Firmware PR is tagged as 4.3.2 not 4.4

Comment (Asizon) [reply src/js/tabs/pid_tuning.js:495]: Ok so, we need to fix it first, it is taged after merge and is merged on ly in master

Comment (haslinghuis) [reply src/js/tabs/pid_tuning.js]: Fixed.

**Review by blckmn — APPROVED** [review]

**Pull request merged**
